packages/core-components/src/components/Table/Table.tsx

```diff
@@ -365,6 +365,15 @@ export function Table<T extends object = {}>(props: TableProps<T>) {
     setSelectedFiltersLength(selectedFiltersArray.flat().length);
   }, [data, selectedFilters, getFieldByTitle]);
 
+  // Check for deprecated checkbox-tree filter
+  useEffect(() => {
+    filters?.map(filter => {
+      // eslint-disable-next-line no-console
+      if (filter.type === 'checkbox-tree')
+        console.warn('"checkbox-tree" filter type is deprecated');
```

Comment:
You won't be permitted to leave out curly braces :) remember to enable automatic prettier at save in your editor.